buildSrc/version.properties

```diff
@@ -20,7 +20,7 @@ httpclient        = 4.5.2
 httpcore          = 4.4.5
 # When updating httpasyncclient, please also update core/src/main/resources/org/elasticsearch/bootstrap/test-framework.policy
 httpasyncclient   = 4.1.2
-commonslogging    = 1.1.3
+commonslogging    = 1.2
```

Comment:
I look at all the uses of commonslogging, and I think that going to 1.2 is safe here with the exception of the Hadoop change which as I mentioned must stay on 1.1.3.